crslab/system/kgsf.py

```diff
@@ -55,6 +56,8 @@ def __init__(self, opt, train_dataloader, valid_dataloader, test_dataloader, voc
         self.rec_batch_size = self.rec_optim_opt['batch_size']
         self.conv_batch_size = self.conv_optim_opt['batch_size']
 
+        self.cpath = os.path.join(CSV_PATH, "kgsf", opt['dataset'])
```

Comment:
```suggestion
        self.csv_path = os.path.join(CSV_PATH, "kgsf", opt['dataset'])
```
`csv_path` 가 분명해서 더 좋지 않을까요?